optgen/cmd/support/framer_gen.go

```diff
@@ -72,10 +72,20 @@ func (g *FramerGen) genNewFramer(def frameDef) {
 		}
 	}
 
+	orderByRequired := def.unit == rang &&
+		((def.start != unboundedPreceding && def.start != startCurrentRow) ||
+			(def.end != unboundedFollowing && def.end != endCurrentRow))
+
+	if orderByRequired {
+		fmt.Fprintf(g.w, "  if len(window.OrderBy.ToExpressions()) != 1 {\n")
```

Comment:
looks like `len(window.OrderBy)` should be equivalent?

Reply:
good call! that helped simplify a little bit.